src/main/java/com/thirdcc/webapp/service/impl/DebtServiceImpl.java

```diff
@@ -55,35 +138,11 @@ public DebtDTO save(DebtDTO debtDTO) {
      */
     @Override
     @Transactional(readOnly = true)
-    public Page<DebtDTO> findAll(Pageable pageable) {
+    public Page<DebtDTO> findAllByEventId(Pageable pageable, Long eventId) {
         log.debug("Request to get all Debts");
+        //TODO implement the method to use getEventAttendeeById
+        //TODO currently will just return all debts
```

Comment:
Why service name findAllByEventId but doing jobs to find all debts?

Reply:
sorry, is implementing it temporary, need to wait for #58 to merge only can use the method